main.py

```diff
@@ -186,7 +186,6 @@ def ui_access_wrapper(self, *args, **kwargs):
     get_redirect_route(r'/console_errors', pages.ConsoleErrorPage),
     get_redirect_route(r'/contact', pages.ContactPage),
 
-    get_redirect_route(r'/forum', pages.ForumPage),
```

Comment:
Maybe leave the forum url here but have the controller do a redirect to the actual google group.

Reply:
Re-added! (Done!)